browserdebuggertools/wssessionmanager.py

```diff
@@ -147,7 +124,7 @@ def run(self):
                 self._empty_send_queue()
```

Comment:
Do we think it is OK to
- not bound the deque (to bound the memory footprint / avoid memory leaks)?
- make `_empty_send_queue` potentially starve/block `_empty_websocket`? (can the usual app or a malicious one manage to put messages in the send queue faster than you can `ws.send` them, so the producer will be stuck in the sending loop?). You could eg copy the send queue before you loop on it, to make sure you only loop on the snapshot.
- make `_empty_websocket` potentially starve/block `_empty_send_queue`? (can the producer be stuck receiving on the ws, and/or take a long time processing the message?). 
- `blocked` + `health_check` may make some of the above scenarios less painful (but I don't think it solves the case of an ever growing send queue), but even then we may be slowing sends/receives that could otherwise happen faster.

So, more generally, are we happy to not send/receive on separate threads? Do we gain more than we lose by it?
(also see the suggestion in `blocked` docstring about separate threads; do we want to update that docstring?)

Reply:
We've discussed the above point and decided to address the send/recv blocking each other in another story since it's very unlikely to happen and not something introduced here.